src/Entity/Phone.php

```diff
@@ -0,0 +1,550 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\PhoneRepository;
+use DateTimeImmutable;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * @ORM\Entity(repositoryClass=PhoneRepository::class)
+ */
+class Phone
+{
+    /**
+     * @var int id
+     *
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer", name="bm_id")
+     */
+    private int $id;
```

Comment:
Codacy found an issue: [Avoid variables with short names like $id. Configured minimum length is 3.](https://app.codacy.com/gh/CarolineDirat/P7APIBileMo/pullRequest?prid=6297993)